src/containers/Editor/LiveEditor/helpers.ts

```diff
@@ -46,6 +44,24 @@ export function getEdgeNodes(
   };
 }
 
+
+export function searchSubTree (trees: any[], id: string) {

```

Comment:
Extremely simple function to search a match `id`, and take a subtree from the JSON data.